am-lib/src/main/java/uk/gov/hmcts/reform/amlib/models/RolePermissions.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.amlib.models;
+
+import com.fasterxml.jackson.core.JsonPointer;
+import lombok.Builder;
+import lombok.Data;
+import uk.gov.hmcts.reform.amlib.enums.AccessType;
+import uk.gov.hmcts.reform.amlib.enums.Permission;
+import uk.gov.hmcts.reform.amlib.enums.SecurityClassification;
+
+import java.util.Map;
+import java.util.Set;
+
+@Data
+@Builder
+public final class RolePermissions {
+    private final Map<JsonPointer, Set<Permission>> permissions;
```

Comment:
As per requirement we are returning this but we are not returning relationships (Claimant or defendant)

Reply:
relationships is only relevant to explicit access. This deals with role based access. Am I missing something?

Reply:
Nope, you are right @swalker125